### PR TITLE
74 provide necessary endpoints for the speaker dashboard

### LINKED ISSUE #76: add endpoint to get all visible events for a a speaker or team member

### PR DESCRIPTION
**Pull request by mgerhold**
I think we have to play around with this implementation as soon as you implemented the frontend. I think there may be some issues lurking in the shadows 🙈

fixes #76

### CODE REVIEW

**Comment by PurpurTentakel97** [comment]
Should I then use this branch to develop the front end and we merge together later on?
What to you think?

**Comment by mgerhold** [comment]
> Should I then use this branch to develop the front end and we merge together later on? What to you think?

Sounds good, let's do it like that 👍

File: app/Controllers/Globals.php [hunk 1]
```diff
@@ -5,6 +5,8 @@
 use App\Controllers\BaseController;
 use App\Models\EventModel;
 use App\Models\GlobalsModel;
+use App\Models\SocialMediaTypeModel;
+use App\Models\SpeakerModel;
```
Comment (PurpurTentakel97) [inline app/Controllers/Globals.php:9]: What are you?

**Review by PurpurTentakel97 — APPROVED** [review]

File: app/Controllers/Globals.php (hunk 1 above)
Comment (mgerhold) [reply app/Controllers/Globals.php]: Removed the redundant imports.

**Review by PurpurTentakel97 — DISMISSED** [review]

**Review by PurpurTentakel97 — APPROVED** [review]

**Pull request merged**